### DISCUSSION TITLE
i am looking for evolving weapons

### DISCUSSION THREAD

**Discussion by alavaelian** [redeclipse/base#1251] [Help and Support]
i was playing long time ago in a server.
after playing for a while the weapons get a second level improving its damage and some cool things like the flamethrower become 
wash thrower and i never find that server again the map was the hospitalm, in the server were only boots and me but i was looking for that server since that day

**Comment by qreeves** [comment]
I didn't even know there was a server like that. You can change the variables whenever you want though, maybe someone was just playing with them?

**Comment by dc-redeclipse** [comment]
I really don't know which server you mean but making vars like that is not a big problem. You can make it on your own by changing the behavior of the flamer by changing the attributes of it. Try modify `/flamercollide.`.. `/flamercook`... ect.
